src/qmapshack/gis/wpt/CGisItemWpt.cpp

```diff
@@ -285,6 +286,30 @@ void CGisItemWpt::newWpt(const QPointF& pt, const QString &name, const QString &
         wpt->setDescription(desc);
     }
 
```

Comment:
Why did you remove 

```
wpt->editInitial();
```

This is changing the expected behavior for adding waypoints.

Reply:
I did not remove it, I moved into a `if` which defaults to true

Reply:
It's missing for 
```
void CGisItemWpt::newWpt(const QPointF& pt, const QString &name, const QString &desc, IGisProject * project)
```

Just compare QMS from `dev` and your branch. The one on dev opens the edit dialog on a new waypoint. Your branch simply sets the waypoint.

Reply:
Ok, my mistake. The way the side by side diff pulled it apart I didn't notice that it was only present in one of the methods :laughing:

Reply:
no worries, all good :D